js/navigation/config.js

```diff
@@ -1,11 +1,15 @@
 import { colors, typography } from '../config/styles';
+import { Image, View } from 'react-native';
+import React from 'react';
+import BackArrow from '../components/BackArrow';
 
 export const sharedNavigationOptions = () => ({
-  headerBackTile: null,
+  headerBackTitle: null,
   headerStyle: {
     backgroundColor: colors.teal,
     height: 65,
   },
+  headerBackImage: <BackArrow />,
```

Comment:
BackArrow does not need to be a component since you are not reusing it. You can write your JSX here.